configs/phalcon.json

```diff
@@ -20,12 +11,21 @@
       "global": true,
       "default_value": "Documentation"
     },
-    "lvl1": {
-      "selector": "???"
-    },
-    "lvl2": ".DocsContainer article h1",
-    "lvl3": ".DocsContainer article h2",
-    "lvl4": ".DocsContainer article h3",
+    "lvl1": ".DocsContainer article h1",
+    "lvl2": ".DocsContainer article h2",
+    "lvl3": ".DocsContainer article h3",
+    "lvl4": ".DocsContainer article h4",
+    "lvl5": ".DocsContainer article h5",
     "text": ".DocsContainer article p, .DocsContainer article li"
-  }
-}
+  },
+  "selectors_exclude": [
+    ".article-menu"
+  ],
+  "js_render": true,
+  "custom_settings": {
```

Comment:
Where is the version defined?

Reply:
meta

Reply:
Oh, so they updated their website. Got it.

Reply:
In their title, they repeat the lvl 0

Reply:
> In their title, they repeat the lvl 0

I don't understand what you mean by that? You mean the `<title>` tag?

Reply:
The time I have cheked it was builded this way: lvl1= Getting started: Introduction (lvl0=Getting started)